mmv1/provider/template_data.go

```diff
@@ -51,6 +51,7 @@ var TemplateFunctions = template.FuncMap{
 	"camelize":   google.Camelize,
 	"underscore": google.Underscore,
 	"contains":   strings.Contains,
+	"join":       strings.Join,
```

Comment:
Sorry, I just merged my [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/10460), which added this method. But no conflict is detected and reported in this PR.

Reply:
No worries. I rebased just in case.